castle/cms/upgrades/upgrade_2_6_31.py

```diff
@@ -0,0 +1,10 @@
+from Products.CMFCore.utils import getToolByName
```

Comment:
I don't think this file is necessary anymore?

Reply:
it might have been old from me, idk. Let me know if i'm overlooking something, though